src/billboard.py

```diff
@@ -0,0 +1,7 @@
+"""Implementation of Name on Billboard Kata on CodeWars"""
+from numpy import multiply
+
+
+def billboard(name, price=30):
+    """Return cost of printing name on billboard."""
+    return multiply(len(name), price)
```

Comment:
The kata specified to not use the `*` operator so I used numpy instead. I think this is not what the kata author had in mind though.